### DISCUSSION TITLE
[Feature] Location Favorites

### DISCUSSION THREAD

**Discussion by miasinterestinglife** [immich-app/immich#9191] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When changing the location an image was taken, you should also be able to setup some location favorites, where you take a lot of pictures so you don't need to painstakingly zoom into the map every single time. This could be in the form of a dropdown menu.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by mbuzina** [comment]
This would massively improve the useability of the location utility. Right now I am plowing through my assets with a notepad page full of coordinates that I copy & paste into the change location dialog view. Just having a favourites list would make that much easier. It does not need to be popluated automatically, just maintain a simple list of names & coordinates.

For an enhanced version, these names could be used instead of / together with the reverse geo lookup resulting in assets located "Home" "At my sisters" instead of (better in addition to) city names. Feed this into the smart search learning and you can search by pictures at my sisters place.

**Reply by rwjorgen64** [reply]
Agreed! Create a list of frequently used locations with a simple name, and search for the simple name for the location (e.g. "Childhood Home")

**Reply by sbopt** [reply]
I suggested something like this in a seperate thread. A location hash-tag would be convenient
#home #work #beach

**Comment by mbuzina** [comment]
See also #22804

**Comment by YarosMallorca** [comment]
Attempted to implement it in #24537

**Comment by mylbeee** [comment]
This would indeed be a great feature to have! Searching the map is too hard for each photo, and the locations won't be exactly the same between photos. Copy-paste of lat/long values is not really an option.

I know you can geotag multiple pictures at once, but in my work flow I'd rather tag the one by one when editing the date and people tags.
